lesson_1.js

```diff
@@ -0,0 +1,96 @@
+//1
+/*const loop = function (times = 0, callback = null){
+for ( let i = 0; i < times; i ++)
```

Comment:
```suggestion
if (typeof callback === 'function') {
for ( let i = 0; i < times; i ++)
```
нужна проверка на передачу функции